app/bundles/PluginBundle/Views/Integration/form.html.twig

```diff
@@ -0,0 +1,194 @@
+{#
+  Variables
+     - form
+     - description
+     - formSettings
+     - formNotes
+     - callbackUrl
+     - activeTab
+     - formThemes (array)
+         May include one or more form themes that need to be applied
+#}
+{%- form_theme form with formThemes -%}
+<!-- form themes: {{ formThemes|join(', ') }} -->
+{#
+if (!$hasSupportedFeatures = (isset($form['supportedFeatures']) && count($form['supportedFeatures']))) {
+    if (isset($form['supportedFeatures'])) {
+        $form['supportedFeatures']->setRendered();
+    }
+}
+#}
+{%- set nSupportedFeatures = form.supportedFeatures is defined ? form.supportedFeatures|length : 0 -%}
+{%- set hasSupportedFeatures = nSupportedFeatures > 0 -%}
+{#{%- if not hasSupportedFeatures and form.supportedFeatures is defined %}{% do form.supportedFeatures.setRendered() %}{% endif -%}#}
+
+{#
+if (!$hasFields = (!empty($formSettings['dynamic_contact_fields']) || isset($form['featureSettings']) && count($form['featureSettings']['leadFields']))) {
+    // Unset if set to prevent features tab from showing when there's no feature to show
+    unset($form['featureSettings']['leadFields']);
+} else {
+    $hideContactFieldTab = (!empty($formSettings['dynamic_contact_fields']) && !count($form['featureSettings']['leadFields']));
+}
+#}
```

Comment:
This might be important. Why is it (and other code) commented out?

Reply:
As I convert the files, I will sometimes comment out the more complicated PHP code to make sure the logic is correct when I run into issues. I'll come back and remove all this and looks like I over looked this. Comments have been removed.